### PR TITLE
Monthly stats page new function

### PR DESCRIPTION
**Pull request by JasonRenBang**
Implement a monthly stats page.

### CODE REVIEW

File: blotztask-ui/package.json [hunk 1]
```diff
@@ -76,5 +77,6 @@
     "prettier": "^3.3.3",
     "tailwindcss": "^3.4.6",
     "typescript": "^5"
-  }
+  },
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```
Comment (Ben0189) [inline blotztask-ui/package.json:81]: why do you change package manager

File: blotztask-ui/yarn.lock
Comment (Ben0189) [inline blotztask-ui/yarn.lock:1]: please remove your yarn lock

File: blotztask-ui/src/app/monthly-stats/page.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+'use client';
+
+import { H1, H5, P } from '@/components/ui/heading-with-anchor';
+import { mock } from 'node:test';
```
Comment (Ben0189) [inline blotztask-ui/src/app/monthly-stats/page.tsx:4]: please remove if it is not used

File: blotztask-ui/src/app/monthly-stats/page.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  ];
+
+  const taskCount = mockTasks.length;
+  return (
+    <>
+      <div className="flex flex-col gap-5 md:items-center md:p-28">
+        <div className="flex flex-col gap-5 ">
+          <img
+            src="/assets/images/profileImage.png"
+            alt="an incredable image"
```
Comment (Ben0189) [inline blotztask-ui/src/app/monthly-stats/page.tsx:34]: try use image attribute

File: blotztask-ui/src/app/monthly-stats/page.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            className="w-[120px] h-[120px] absolute top-[80px] left-[500px]"
+          />
+          <p className="font-arial font-bold text-[40px] leading-[100px] tracking-[-0.41px] text-center text-[#278291]">
```
Comment (Ben0189) [inline blotztask-ui/src/app/monthly-stats/page.tsx:37]:
```suggestion
   text-secondary
```

File: blotztask-ui/src/app/page.tsx [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
-        providing functionality for task creation, management, and completion
-        tracking.
-      </H3>
+      <p className="w-[679px] h-[60px] absolute top-[197px] left-[402px] font-arial text-[32px] font-bold leading-[48px] tracking-[-0.41px] text-center text-[#278291]">
+        Welcome to the new age note taking web
+      </p>
+      <img
+        src="/assets/images/homePageNewPicture.png"
+        alt="an incredable image"
+        className="w-[849px] h-[350px] absolute top-[305px] left-[296px]"
```
Comment (Ben0189) [inline blotztask-ui/src/app/page.tsx:12]: dont use absolute

File: blotztask-ui/src/app/task-dayview/page.tsx [hunk 6]
```diff
@@ -58,24 +58,25 @@ export default function Dayview() {
     // Simulate fetching tasks
     setTasks(validatedTasks);
   }, []);
-
+  // Count this month's tasks' number for user
+  const countTasks = (tasks: taskDto[]): number => {
```
Comment (Ben0189) [inline blotztask-ui/src/app/task-dayview/page.tsx:62]: why do you need to change the day view page

**Review by Ben0189 — CHANGES_REQUESTED** [review]

**Review by Ben0189 — APPROVED** [review]

**Comment by JasonRenBang** [comment]
all good

**Pull request merged**